pages/common/shift.md

```diff
@@ -0,0 +1,12 @@
+# shift
+
+> A bash built-in command that shifts the (numbered) arguments of a function or script by a specified number of places.
+> See <https://www.gnu.org/software/bash/manual/bash.html> for more information.
+
+- Move arguments by one place dropping the first argument:
+
+`shift`
+
+- Move arguments by {{n}} places dropping the first {{n}} arguments:
+
+`shift {{n}}`
```

Comment:
We don't typically use token syntax inside example descriptions.  I'd do this, in this case:
```suggestion
- Move arguments by N places dropping the first N arguments:

`shift {{N}}`
```